gridcomps/Cap/MAPL_CapGridComp.F90

```diff
@@ -1103,146 +1128,192 @@ subroutine run_MAPL_GridComp(gc, rc)
   end subroutine run_MAPL_GridComp
 
 
-  subroutine step(this, rc)
+  subroutine step(this, unusable, phase, rc)
     class(MAPL_CapGridComp), intent(inout) :: this
-    integer, intent(out) :: rc
+    class(KeywordEnforcer), optional, intent(in   ) :: unusable
+    integer, optional, intent(in) :: phase
+    integer, optional, intent(out) :: rc
+
     integer :: AGCM_YY, AGCM_MM, AGCM_DD, AGCM_H, AGCM_M, AGCM_S
     integer :: status
 ! For Throughout/execution estimates
-      integer           :: HRS_R, MIN_R, SEC_R
-      real(kind=REAL64) :: START_RUN_TIMER,END_RUN_TIMER,START_TIMER,END_TIMER
-      real(kind=REAL64) :: TIME_REMAINING
-      real(kind=REAL64) ::  LOOP_THROUGHPUT=0.0_REAL64
-      real(kind=REAL64) ::  INST_THROUGHPUT=0.0_REAL64
-      real(kind=REAL64) ::   RUN_THROUGHPUT=0.0_REAL64
-      real              :: mem_total, mem_commit, mem_committed_percent
-      real              :: mem_used, mem_used_percent
+    integer           :: HRS_R, MIN_R, SEC_R
+    real(kind=REAL64), save :: START_RUN_TIMER,END_RUN_TIMER,START_TIMER,END_TIMER
```

Comment:
Sorry - one more change.     We don't want SAVE here - just to dangerous.  Create a small derived type for these items and then add a component of that type into the CapGridComp type.

Reply:
I agree with Tom, 'save' could have implications (especially if we are going to run multi-threaded)

Reply:
Good point.  Changed